examples/block-explorer/explorer-indexer/explorer_indexer.manifest.yaml

```diff
@@ -7,6 +7,6 @@ graphql_schema: examples/block-explorer/explorer-indexer/schema/explorer_indexer
 module:
   wasm: target/wasm32-unknown-unknown/release/explorer_indexer.wasm
 metrics: ~
-contract_id: ~
+contract_id: ~ 
```

Comment:
Seems that a space was added here. That's not required for the code to work properly, right?

Reply:
Nope, just a typo. Will fix.